src/stm32/stm32g0.c

```diff
@@ -56,7 +60,7 @@ lookup_clock_line(uint32_t periph_base)
         return (struct cline){.en=&RCC->APBENR2,.rst=&RCC->APBRSTR2,.bit=1<<18};
     if (periph_base == ADC1_BASE)
         return (struct cline){.en=&RCC->APBENR2,.rst=&RCC->APBRSTR2,.bit=1<<20};
-    if (periph_base >= APBPERIPH_BASE && periph_base <= LPTIM1_BASE)
+    if (periph_base >= APBPERIPH_BASE && periph_base < APBPERIPH_BASE + 0x8000)
```

Comment:
What does this do?  What is 0x8000?

Reply:
0x8000 is the offset from the APB bus base address at which the peripheral after LPTIM1 starts. The issue is that we don't have LPTIM1 on the G0x0, so we can't use that peripheral base address in the if statement.
As a further note, this if statement selects bits from a 32 bit register that controls power where the peripheral address neatly maps to the power bit. As such, at most 32 peripherals can be controlled by that register. Each peripheral is 0x400 in length in memory (see a few lines lower). 0x8000 is 0x400 * 32.
![image](https://user-images.githubusercontent.com/17808203/206990739-8314530e-0055-4e05-8b79-6464eaae19a8.png)